core/meta/src/TStreamerElement.cxx

```diff
@@ -195,9 +195,9 @@ ClassImp(TStreamerElement);
 
 TStreamerElement::TStreamerElement()
 {
-   fType        = 0;
+   fType        = TVirtualStreamerInfo::kUnset;
```

Comment:
IIUC here the value is changed from `0` to `-3`, I believe this is a conscious choice but I wanted to make sure that we know it won't lead to unintended behaviour elsewhere (e.g. some function expecting `fType == 0`).